### ISSUE TITLE
Class constants are not documented when a with statement is present

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1102]
Steps to reproduce:

```
$ git clone git@github.com:clee704/sphinxbug.git
$ mkvirtualenv sphinxbug
$ make init
$ make docs
```

The class constant `CONSTANT` is not documented while it should. It seems to be caused by [the with statement](http://docs.python.org/2/reference/compound_stmts.html#the-with-statement) with multiple variables.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1102
- Originally reported by: Anonymous
- Originally created at: 2013-02-10T14:49:35.315

**Comment by shimizukawa** [comment]
_From Choongmin Lee on 2013-02-10 13:53:15+00:00_

I found I can workaround this issue by using nested with statements.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-30 16:45:46+00:00_

Closes #1102: Support multi-context "with" statements in autodoc by updating
pycode grammar from lib2to3.

→ <<cset 2e8b8ba7e33d4c55d26e32de02ae38aa7ae48ddc>>
